api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -101,7 +114,7 @@ void createLineageEvent(
       String producer);
 
   @Transaction
-  default void updateMarquezModel(LineageEvent event) {
+  default UpdateLineageResponse updateMarquezModel(LineageEvent event) {
```

Comment:
Do we want a service to return a db model? Meaning, shouldn't the service define its own model to return and not depend on a model defined by a layer it encapsulates?

Reply:
I'll adjust it to be the bag of objects like we discussed before and just do the unboxing in the returning function call.